src/platform/silabs/CHIPPlatformConfig.h

```diff
@@ -106,6 +106,10 @@
 #define CHIP_CONFIG_ICD_IDLE_MODE_DURATION_SEC SL_IDLE_MODE_DURATION_S
 #endif // CHIP_CONFIG_ICD_IDLE_MODE_DURATION_SEC
 
+#ifndef CHIP_CONFIG_ICD_SHORT_IDLE_MODE_DURATION_SEC
```

Comment:
Could you add one example configuration with explicit comments for both silabs and linux in LIT?
For example, here?
https://github.com/project-chip/connectedhomeip/blob/master/examples/lit-icd-app/silabs/openthread.gni